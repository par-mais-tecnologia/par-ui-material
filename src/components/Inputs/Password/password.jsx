import React, { PureComponent } from 'react'
import TextFieldMUI from '@material-ui/core/TextField'
import withStyles from "@material-ui/core/es/styles/withStyles";

const style =  {
    marginTop: '13px'
}

class Password extends PureComponent {

    render () {
        return (
                <TextFieldMUI
                    inputProps={{style: style}}
                    {...this.props} />
            )
        }
    }

export default withStyles(style)(Password);