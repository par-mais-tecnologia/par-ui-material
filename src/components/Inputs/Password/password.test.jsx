import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Password from './password'

configure({ adapter: new Adapter() })

export const test = describe('Password', () => {
    it('shallow Password', () => {
        const wrapper = shallow(
            <Password />
        )
        expect(wrapper).toBeTruthy()
    })

    it('shallow Password match Snapshot', () => {
        const wrapper = shallow(
            <Password />
        )
        expect(wrapper).toMatchSnapshot()
    })
})
