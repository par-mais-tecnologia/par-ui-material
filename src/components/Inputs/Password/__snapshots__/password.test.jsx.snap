// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Password shallow Password match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Password />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "required": false,
      "select": false,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "required": false,
        "select": false,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
